src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalFinalVariableNameCheckTest.java

```diff
@@ -59,6 +59,8 @@ public void testDefault()
 
         final String[] expected = {
             "123:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
+            "194:29: " + getCheckMessage(MSG_INVALID_PATTERN, "BR", pattern),
```

Comment:
See my comment at https://github.com/checkstyle/checkstyle/pull/4116#discussion_r108506898 which I asked you to change but you didn't.

> We are not expecting changes to this file.
Changes for your issue should be in an input file that no other tests uses to avoid this conflict.